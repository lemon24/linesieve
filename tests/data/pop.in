--section '(\S+):$'
push t
  push o
    sub 1 2
  pop
  sub '(\d)' '\1\1'
---
one:
1
two:
1
three:
1
four:
1
five:
1
