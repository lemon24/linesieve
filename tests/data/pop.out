one
2
two
22
three
22
four
2
five
1
